src/de/mod.rs

```diff
@@ -2955,12 +2989,12 @@ mod tests {
                 vec![
                     // This comment here to keep the same formatting as others
                     // otherwise rustfmt suggest one-line it
-                    Text(Cow::Borrowed("text")),
+                    Text("text".into()),
                     End(BytesEnd::new("inner")),
                 ]
             );
             assert_eq!(de.write, vec![]);
-            assert_eq!(de.next().unwrap(), Text(Cow::Borrowed("text")));
+            assert_eq!(de.next().unwrap(), Text("text".into()));
```

Comment:
I'm assuming we don't care whether it was borrowed or owned?  (not that this aspect is being properly tested currently)

Reply:
Yes, we don't care. Only the fact that `Text` event is generated is tested here